src/api/VoteMonitor.Api.Form/DependencyInjectionExtensions.cs

```diff
@@ -9,10 +9,10 @@ public static class DependencyInjectionExtensions
     {
         public static IServiceCollection AddFormServices(this IServiceCollection services)
         {
-            services.AddScoped<IUpdateOrCreateEntityMapper<Entities.Form, FormDTO>, HierarchicalMapper<Entities.Form, FormDTO, FormSection, FormSectionDTO>>();
-            services.AddScoped<IUpdateOrCreateEntityMapper<FormSection, FormSectionDTO>, HierarchicalMapper<FormSection, FormSectionDTO, Question, QuestionDTO>>();
-            services.AddScoped<IUpdateOrCreateEntityMapper<Question, QuestionDTO>, HierarchicalMapper<Question, QuestionDTO, OptionToQuestion, OptionToQuestionDTO>>();
-            services.AddScoped<IUpdateOrCreateEntityMapper<OptionToQuestion, OptionToQuestionDTO>, UpdateOrCreateEntityMapper<OptionToQuestion, OptionToQuestionDTO>>();
```

Comment:
OptionToQuestion mapper still needs to be registered but as `IEntityMapper`. Currently, I'm getting an error while starting the app related to the DI